src/registers.ts

```diff
@@ -0,0 +1,18 @@
+import { getKonfluxPublicInfo } from '~/hooks/useKonfluxPublicInfo';
+import { KonfluxInstanceEnvironments } from '~/types/konflux-public-info';
+import { registerCondition } from './feature-flags/conditions';
+import { checkIfKubeArchiveIsEnabled } from './kubearchive/conditional-checks';
+
+registerCondition('isKubearchiveEnabled', checkIfKubeArchiveIsEnabled);
+
+registerCondition('isStagingCluster', async () => {
+  try {
+    const info = await getKonfluxPublicInfo();
+    return info.environment === KonfluxInstanceEnvironments.STAGING;
+  } catch (error) {
+    return false;
```

Comment:
In case, we are not able to determine the staging environment should we return true here? If we return false then the flag will show up on all environments, returning true will prevent it from showing up on any environment.